gobblin-runtime/src/main/java/org/apache/gobblin/runtime/api/MysqlMultiActiveLeaseArbiter.java

```diff
@@ -199,23 +197,27 @@ public LeaseAttemptStatus tryAcquireLease(DagActionStore.DagAction flowAction, l
       int leaseValidityStatus = resultSet.getInt(4);
```

Comment:
hopefully doesn't feel like overkill, but I'd abstract this by defining a `static` inner `@Data` class with an overloaded constructor (or `static` factory method) taking a `ResultSet`

Reply:
I don't see a large benefit from this since the static class will have to encode these column retrievals anyway but I made it a bit more clear by using the column name instead of index to retrieve the values so it's more readable.